### PR TITLE
remove New*** from variable name.

### PR DESCRIPTION
**Pull request by hanfei1991**
@shenli @coocood @zimulala @tiancaiamao @XuHuaiYu PTAL

### CODE REVIEW

File: executor/aggregate_xapi_test.go [hunk 1]
```diff
@@ -13,483 +13,484 @@
 
 package executor
```
Comment (ngaut) [inline executor/aggregate_xapi_test.go]: Looks like it's useless. Remove the file.

Comment (shenli) [inline executor/aggregate_xapi_test.go]: @hanfei1991 Please remove this file and add tests for NewXAggregateExec in this PR or in the  next PR.

**Comment by shenli** [comment]
Rest LGTM

File: executor/aggregate_xapi_test.go (hunk 1 above)
Comment (hanfei1991) [inline executor/aggregate_xapi_test.go]: ok, I will add tests for the next PR.

**Comment by shenli** [comment]
LGTM
PTAL @zimulala @coocood @tiancaiamao

**Comment by zimulala** [comment]
LGTM

**Pull request merged**